conman/routes/utils.py

```diff
@@ -13,7 +13,7 @@ def split_path(path):
     path = path.rstrip('/')
 
     while path:
-        paths.append(path + '/')
+        paths.insert(1, path + '/')
         path = os.path.split(path)[0]
         if path == '/':
             break
```

Comment:
I doubt profiling would reveal any noticeable difference between this and a `deque`, but I suspect it would be a whole lot easier to read. I'll pop a `deque` in there, thanks for the suggestion `:)`